scalding-commons/src/main/java/com/twitter/scalding/commons/scheme/KeyValueByteScheme.java

```diff
@@ -24,6 +26,11 @@ public KeyValueByteScheme(Fields fields) {
     super(fields, BytesWritable.class, BytesWritable.class);
   }
 
+  @Override
+  public void sourceConfInit(FlowProcess<JobConf> flowProcess, Tap<JobConf, RecordReader, OutputCollector> tap, JobConf conf) {
+    conf.setInputFormat(CombineSequenceFileInputFormat.class);
```

Comment:
Hadoop has better community support, EB used only inside Twitter.

@isnotinvain / @johnynek what do you think?